### PR TITLE
get the pullsecret from MCH CR

### PR DESCRIPTION
**Pull request by hchenxa**
Signed-off-by: hchenxa <huichen@redhat.com>

### CODE REVIEW

File: pkg/utils/mco_deploy.go [hunk 1]
```diff
@@ -790,24 +797,58 @@ func CheckMCOConversion(opt TestOptions, v1beta1tov1beta2GoldenPath string) erro
...
+		return err
+	}
+
 	pullSecret, errGet := clientKube.CoreV1().Secrets(mcoNs).Get(name, metav1.GetOptions{})
 	if errGet != nil {
 		return errGet
 	}
 
 	pullSecret.ObjectMeta = metav1.ObjectMeta{
-		Name:      name,
```
Comment (clyang82) [inline pkg/utils/mco_deploy.go:806]: Could you keep using the name fetched from mch to update the mco cr?

**Review by morvencao — COMMENTED** [review]
/lgtm

**Pull request merged**